README.MD

```diff
@@ -2,61 +2,47 @@
 
 # Project Name
 
-> Description the project.
+> Capstone project created using [this](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) design template.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- Javascript
 
 ## Live Demo (if available)
```

Comment:
- [ ] Kindly remove (if available) its not needed.

Reply:
Done